gravity.sh

```diff
@@ -574,6 +575,10 @@ parseList() {
   else
     echo "  ${INFO} Imported ${num_domains} domains"
   fi
+
+  # remove temp_file
+  rm "${temp_file}" > /dev/null 2>&1 || \
```

Comment:
A less fragile way to do this is to create the `temp_file` open a file handle to it and then immediately `rm` it. Use the file handle as the access point. We do this in the debug process for security but it's also useful in that it doesn't leave a file around if the process terminates for what ever reason.

https://github.com/pi-hole/pi-hole/blob/master/advanced/Scripts/piholeDebug.sh#L194